package.json

```diff
@@ -42,12 +42,6 @@
       "@commitlint/config-conventional"
     ]
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "kathari precommit",
-      "commit-msg": "kathari commitmsg"
-    }
-  },
```

Comment:
This might have been sloppy from me. I was getting some strange errors when committing the changes and removing this solved the issue. I think that I copied it form PoH but then I noticed that not all Kleros contracts have it. Should I keep this?

Reply:
They are pre-commit hooks for linting so that we don't push code that fails linting.

Reply:
Ok, sorry about that then. I'll add this back

Reply:
Good thing you mentioned this, there were many things to fix. For the contracts I had to disable some linter features, because apparently neither solhint nor ethlint are yet compatible with solidity 0.7